src/main/java/com/sps/Servlets/NewRecipeServlet.java

```diff
@@ -58,7 +59,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     Entity recipeEntity = null;
     try {
       recipeEntity = datastore.get(KeyFactory.stringToKey(keyString));
-    } catch (EntityNotFoundException e) {
+    } catch (Exception e) {
```

Comment:
Did you get into more exceptions? Can you document the types of exceptions that is possible to happen in this case?

Reply:
In all honesty, I'm not sure why I made this change.

Reply:
Never mind, I see why. stringToKey() can also throw an exception if the keyString isn't parsable, so I think I made the catch block use a global exception to catch all errors. I'll split it into its two error cases.

Reply:
Actually, because both exceptions result in the same behavior, I left them in one global Exception catch block, and added clarifying javadoc to the method.